css/styles.css

```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 1.6;
+    color: #2b2b2b;
+    background-color: #f5f5f5;
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    margin: 0;
+    /* Put content in horizontal center */
+    text-align: center;
+    /* Put content in vertical center */
+    justify-content: center;
+}
+
+p {
+    /* Add space around beginning and end of text */
+    margin-inline: 15px;
+}
+
+.header__logo {
+    max-width: 100px;
+}
```

Comment:
Great applying what you learned in the previous module. I think it helps concepts stick. 👍🏾